src/utiltime.cpp

```diff
@@ -25,7 +28,7 @@ int64_t GetTime()
 
 void SetMockTime(int64_t nMockTimeIn)
 {
-    nMockTime = nMockTimeIn;
+    nMockTime.store(nMockTimeIn, std::memory_order_relaxed);
```

Comment:
It's read by a relaxed read anyway, using a stronger serializing write won't matter (if I understand the semantics correctly).